llvm/lib/Transforms/Instrumentation/InstrProfiling.cpp

```diff
@@ -148,6 +150,16 @@ cl::opt<bool> SkipRetExitBlock(
     "skip-ret-exit-block", cl::init(true),
     cl::desc("Suppress counter promotion if exit blocks contain ret."));
 
+static cl::opt<bool>
+    SampledInstrument("sampled-instr", cl::ZeroOrMore, cl::init(false),
```

Comment:
naming nit: SampledInstr to be consistent with command name. Similar change in other places.

Reply:
OK. Will change.